service/worker/scheduler/testdata/generate_history.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -x
+
+id=sched1
+
+trap "temporal schedule delete -s $id" EXIT
+
+temporal schedule create -s $id \
+  --overlap-policy bufferall \
+  --interval 10s \
+  --jitter 8s \
```

Comment:
Why do we need the jitter?

Reply:
mainly because the change I'm making, which I wanted to use replay testing to verify, concerns jitter.
ideally this would exercise all the features of the workflow, this is just a quick starting point to get something out there.